src/components/PersonalInfoCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ActionButton from 'components/ActionButton';
+import TextInput from 'components/TextInput';
+import { Title, Card } from './Card';
+
+const PersonalInfoContainer = styled(Card)`
+	grid-area: personal;
+`;
+
+const Content = styled.div`
+	margin: 2.5em;
+	display: grid;
+	grid-template-columns: 15em 2fr 2fr 2fr;
+	grid-template-rows: auto;
+	grid-template-areas:
+		'title title title title'
+		'avatar Col1 Col2 .';
+`;
+
+const PersonalInfoTitle = styled(Title)`
+	margin-top: 0;
+`;
+
+const ProfilePicture = styled.div`
+	font-size: 10em;
+	padding-left: 0.2em;
+	margin-right: 0.2em;
+`;
+const Col1 = styled.div`
+	grid-area: Col1;
+	padding-right: 2em;
+`;
```

Comment:
You can make this into one component and accept a prop

```suggestion
const Col = styled.div<{ gridArea }>`
	grid-area: ${(props) => props.gridArea};
	padding-right: 2em;
`;
```

This can replace Col1 and Col2